core/infra/protobuf/api.pb.cc

```diff
@@ -340,10 +340,11 @@ void protobuf_AssignDesc_api_2eproto() {
       GOOGLE_PROTOBUF_GENERATED_MESSAGE_FIELD_OFFSET(Transaction, _internal_metadata_),
       GOOGLE_PROTOBUF_GENERATED_MESSAGE_FIELD_OFFSET(Transaction, _is_default_instance_));
   ConsensusEvent_descriptor_ = file->message_type(14);
-  static const int ConsensusEvent_offsets_[3] = {
+  static const int ConsensusEvent_offsets_[4] = {
```

Comment:
this thing will be executed during configure step (e.g. `cmake ..`), what if somebody wants just to read the code and accidentally will find out that some headers are missing?